src/runtime/offers-flow-test.js

```diff
@@ -310,46 +310,104 @@ describes.realWin('OffersFlow', (env) => {
     await offersFlow.start();
   });
 
-  it('passes isClosable to openView when false for new offer flows', async () => {
-    const clientConfigManager = runtime.clientConfigManager();
-    sandbox.stub(clientConfigManager, 'getClientConfig').resolves(
-      new ClientConfig({
-        useUpdatedOfferFlows: true,
-      })
-    );
-    offersFlow = new OffersFlow(runtime, {'isClosable': false});
-    dialogManagerMock
-      .expects('openView')
-      .withExactArgs(
-        sandbox.match.any,
-        false,
-        sandbox.match({
-          closeOnBackgroundClick: false,
+  describe('click experiment active', () => {
```

Comment:
Same in this file with setup in before.

Reply:
same with "sandbox is not defined" error.